Target specific blocks for debug logging (#383)
jvalue/jayvee: apps/docs/docs/user/intro.md

jv <file> -d
```

With **runtime parameters**:

```console
jv -e <param>=<value> -e <param>=<value> ... <file>
```

### Debug a `.jv` file

> Currently, we only allow print debugging by the parameters `dg` and `dt`.

Reviewer: ```suggestion
Print debugging is controlled by the parameters `--debug-granularity` and `--debug-target`.
```

Reviewer: Reading this again I think we should always use the long form parameters in the docs, that way they are easier to remember and understand.

Also, I'd never use 'currently' or 'only' in the docs. We can have a roadmap with plans but the docs are a place to describe the current state.

Reviewer: Also, reading this I was initially confused if using `-d` now logs nothing because `skip` is the default for granularity. If I understand right, `skip` in that context means "Skip extra debug output with data for each block". But it reads really confusing. I'd rename `--debug-granularity` to something like `--debug-data` or `skip` to `pipeline-only` (not the best name but the only thing I spontaneously got).

Author: Thx. I agree that naming is quite hard here...
How about `debug-output-granularity` (`dg`) and `debug-output-targets` (`dt`)?

Reviewer: If you have a `debug` flag and call anything a debug granularity of `skip`, I would assume it to skip debug output. I would honestly just rename `skip` to something like `minimal`?
